.github/workflows/build.yaml

```diff
@@ -7,24 +7,20 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
```

Comment:
Oef, the reason for this is because we need to create two FE images for the different environments.
As Tom wasn't able to inject an environmental variable into the FE on run we needed to switch the env's at build time.
That is why we got this structure where the VITE_KEYCLOAK_CLIENT is set on build.
It might be better to create two different build files and have them both be triggered